src/intro-to-c/debugging/gdb-intro.md

```diff
@@ -243,9 +267,11 @@ multiply_by_two (current=0x0, arr=0x7fffffffdfe0) at my_file.c:37
 (gdb)
 ```
 
+> The `until` command lets you continue running your program until you reach a certain line. The program will still stop on any breakpoints.
```

Comment:
I see no explanation on the `step` command. Kind of self-explanatory, but might want to add that in.